omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx.conf.erb

```diff
@@ -141,6 +141,7 @@ http {
           server {
             <%= @helper.listen_port('http') %>
             server_name <%= @helper.server_name %>;
+            more_clear_headers <%= @helper.server_name %>;
```

Comment:
❓ From your commit message, I would expect this to be `more_clear_headers Server;`, but this renders to something else. How come?

Reply:
@srenatus : Sorry I have update that actually on dev I am looking something and that was push please have a look.